src/NuGet.Server/Infrastructure/JsonNetPackagesSerializer.cs

```diff
@@ -21,15 +24,30 @@ public void Serialize(IEnumerable<ServerPackage> packages, Stream stream)
         {
             using (var writer = new JsonTextWriter(new StreamWriter(stream, Encoding.UTF8, 1024, true)))
             {
-                _serializer.Serialize(writer, packages.ToList());
+                _serializer.Serialize(
+                    writer,
+                    new SerializedServerPackages
+                    {
+                        SchemaVersion = CurrentSchemaVersion,
```

Comment:
This code is more about serialization of the package metadata cache. I gave the serialization format a version number so that as the schema iterates, the cache is rebuilt. This is an optimization and concern internal to the server. Both SemVer 1.0 and 2.0 packages will be stored in this metadata cache file.